docs/code_contribution.md

```diff
@@ -4,6 +4,22 @@ To make the EAGER2 code and processing logic more understandable for new contrib
 
 If you wish to contribute a new module, please use the following coding standards.
```

Comment:
Shouldn't all that go into `.github/CONTRIBUTING.md`?

Reply:
Yup, merged the two together now 👍